{"body": "v20, with this fix, was deployed on September 30th. Any data after that should not exhibit this problem.\n", "body_text": "v20, with this fix, was deployed on September 30th. Any data after that should not exhibit this problem.", "url": "https://api.github.com/repos/mozilla/normandy/issues/comments/254294441", "created_at": "2016-10-17T18:37:23Z", "author_association": "MEMBER", "html_url": "https://github.com/mozilla/normandy/issues/277#issuecomment-254294441", "updated_at": "2016-10-17T18:37:23Z", "user": {"following_url": "https://api.github.com/users/mythmon/following{/other_user}", "events_url": "https://api.github.com/users/mythmon/events{/privacy}", "organizations_url": "https://api.github.com/users/mythmon/orgs", "url": "https://api.github.com/users/mythmon", "gists_url": "https://api.github.com/users/mythmon/gists{/gist_id}", "html_url": "https://github.com/mythmon", "subscriptions_url": "https://api.github.com/users/mythmon/subscriptions", "avatar_url": "https://avatars3.githubusercontent.com/u/305049?v=4", "repos_url": "https://api.github.com/users/mythmon/repos", "received_events_url": "https://api.github.com/users/mythmon/received_events", "gravatar_id": "", "starred_url": "https://api.github.com/users/mythmon/starred{/owner}{/repo}", "site_admin": false, "login": "mythmon", "type": "User", "id": 305049, "followers_url": "https://api.github.com/users/mythmon/followers"}, "body_html": "<p>v20, with this fix, was deployed on September 30th. Any data after that should not exhibit this problem.</p>", "id": 254294441, "issue_url": "https://api.github.com/repos/mozilla/normandy/issues/277"}